### PR TITLE
Update automation.md

### PR DESCRIPTION
**Pull request by NickLindeberg**
Updated Automation For 1903 Inning.

### CODE REVIEW

File: m4_sessions/1903_inning/automation.md [hunk 1]
```diff
@@ -1,53 +1,48 @@
...
+As your technical careers progress, most of the work you will do is to automate processes that have been done manually. Our mission at Turing is to “unlock human potential.” The software we write enables it’s users to be more productive, and to direct energy toward solving increasingly difficult problems. However, some automation, especially of physical processes, can force some out of the jobs they’ve done for years. This has the potential to do more harm than good for inclusion in our society.
 
-## Pre-Session
+#### Introductions 1:00 - 1:05
+  * Have students introduce themselves
+    Name:
+    CoHort: 
+    Pronouns:
+    Answer Questions:
+   1. If you could have one process in your life automated tomorrow what would it be?
```
Comment (emhickmann) [inline m4_sessions/1903_inning/automation.md:17]: Great opening

File: m4_sessions/1903_inning/automation.md [hunk 2]
```diff
@@ -1,53 +1,48 @@
...
-  		  
-As your technical careers progress, most of the work you will do is to automate processes that have been done manually. Our mission is to “unlock human potential.” The software we write enables it’s users to be more productive, and to direct energy toward solving increasingly difficult problems. However, some automation, especially of physical processes, can force some out of the jobs they’ve done for years. This has the potential to do more harm than good for inclusion in our society.
+As your technical careers progress, most of the work you will do is to automate processes that have been done manually. Our mission at Turing is to “unlock human potential.” The software we write enables it’s users to be more productive, and to direct energy toward solving increasingly difficult problems. However, some automation, especially of physical processes, can force some out of the jobs they’ve done for years. This has the potential to do more harm than good for inclusion in our society.
 
-## Pre-Session
+#### Introductions 1:00 - 1:05
+  * Have students introduce themselves
+    Name:
```
Comment (emhickmann) [inline m4_sessions/1903_inning/automation.md:13]: Love this

File: m4_sessions/1903_inning/automation.md [hunk 3]
```diff
@@ -1,53 +1,48 @@
...
+   1. If you could have one process in your life automated tomorrow what would it be?
+   2. What automated process today do you appreciate the most?
 
+#### Icebreaker 1:05 - 1:10
+   Automation icebreaker?
 
-### Survey Questions
-Survey to be sent either prior to session or immediately after ( prior may lead to better response? )
+### Groupings 1:10 - 1:15
+  We will group members by opposing stances. Have students stand in a spectrum of how much they like the idea of automation, and from that hand select one each from opposing sides and a middle. This should facilitate the most conversation and opposing viewpoints.
```
Comment (emhickmann) [inline m4_sessions/1903_inning/automation.md:24]: Will it be groups of 3/4?

File: m4_sessions/1903_inning/automation.md [hunk 4]
```diff
@@ -1,53 +1,48 @@
...
-#### Survey to lighten the mood
-  * funny questions like:
-   1. When the robots attack, what’s the first thing you do?
-   2. If you could have one process in your life automated tomorrow what would it be?
-   3. What automated process today do you appreciate the most
-   4. Write your apology to the person whose job you stole by automating that thing.
-   
-### Pre-Reading
+#### Whiteboard Pros and Cons 1:15 - 1:25
+  Have groups write pros and cons on board or papers.
```
Comment (emhickmann) [inline m4_sessions/1903_inning/automation.md:27]: Are they writing pro/con of automation?  Or are they reacting to something? I can have chart paper/markers for you.

File: m4_sessions/1903_inning/automation.md [hunk 5]
```diff
@@ -1,53 +1,48 @@
...
+  Each group shares a pro and con  
+  
+### POM 1:25
 
-Send pre-reading out at least 24 hours before the session
+#### Watch Video 1:30 - 1:45
```
Comment (emhickmann) [inline m4_sessions/1903_inning/automation.md:32]: Which video are they watching here?  The full 15 minutes?  

I would share the questions ahead of the video so they can start thinking about the questions.  Also please turn on closed captioning for the video!

File: m4_sessions/1903_inning/automation.md [hunk 6]
```diff
@@ -1,53 +1,48 @@
...
+#### Answer Questions In Notebook 1:45 - 1:50
 
-*   ["Machines for Thinking"](http://www.economist.com/news/books-and-arts/21669597-computers-will-get-smarter-humans-charge-machines-thinking) (_The Economist_: October 1, 2015)
-*   "The Rise of the Robots by Martin Ford / Humans Need Not Apply by Jerry Kaplan – review"
-https://www.theguardian.com/books/2015/oct/01/the-rise-of-robots-humans-need-not-apply-review (_The Guardian_: October 1, 2015)
-*   ["Jerry Kaplan: Humans Need Not Apply at Google"](https://www.youtube.com/watch?v=JiiP5ROnzw8) (Google: November 12, 2015)
+1) How serious an impact do you think automation will have on our economy from now until you retire? <br> 
+2) Where do you think you fall across the spectrum of people that will be affected by these changes? <br> 
+3) What do you think it means to be a responsible citizen around this issue? <br> 
+4) What role can Turing play in advancing civic responsibility in response to the challenges automation will bring?
```
Comment (emhickmann) [inline m4_sessions/1903_inning/automation.md:39]: What role do you have as a software developer?  What is the danger of not considering these questions?

File: m4_sessions/1903_inning/automation.md [hunk 7]
```diff
@@ -1,53 +1,48 @@
...
-| 13 min | Introduce discussion section: Students should identify a timekeeper, a faciliator, and a notetaker.  Each student 
-should spend about 2 minutes sharing what surfaced during their writing. After individual sharing, the group facilitator can help the group decide what themes from people's writing to discuss in more depth. | Facilitator: walk the room and listen to group discussions.  Identify a few key points or themes from the small group discussions that you might want to share with the whole group |
-
-### A second video will be shown to close out the session.
-  The video will be selected as a more positive ( if possible ) solution or pros to the topic of automation. Video is meant to offer a new perspective and possible change in mood as the session closes.. Hopefully to foster creative thought and less negativity.. or more thoughts on possible solutions.
+#### Share with group 1:50 - 1:55
```
Comment (emhickmann) [inline m4_sessions/1903_inning/automation.md:41]: How are people sharing?  Will one person from each group present?  Will each group share thoughts from one question?

File: m4_sessions/1903_inning/automation.md [hunk 8]
```diff
@@ -1,53 +1,48 @@
...
+
+#### Share with class 1:55 - 1:57
```
Comment (emhickmann) [inline m4_sessions/1903_inning/automation.md:43]: What are you sharing with the class?  Closing comments?  

What do you want students thinking about as they leave this session?

**Review by emhickmann — COMMENTED** [review]
@NickLindeberg Consider what each facilitator is doing during the different sections...who is leading each part?  What role can the facilitator take?  How could you push people's thinking on this topic?

File: m4_sessions/1903_inning/automation.md (hunk 2 above)
Comment (NickLindeberg) [reply m4_sessions/1903_inning/automation.md:13]: We do too.  :-)

File: m4_sessions/1903_inning/automation.md (hunk 1 above)
Comment (NickLindeberg) [reply m4_sessions/1903_inning/automation.md:17]: Thanks!

File: m4_sessions/1903_inning/automation.md (hunk 3 above)
Comment (NickLindeberg) [reply m4_sessions/1903_inning/automation.md:24]: Hopefully four groups.  Size will depend on number of attendees.

File: m4_sessions/1903_inning/automation.md (hunk 4 above)
Comment (NickLindeberg) [reply m4_sessions/1903_inning/automation.md:27]: We would like four charts and +-37 markers.  They will be stating their thoughts about automations.

File: m4_sessions/1903_inning/automation.md (hunk 6 above)
Comment (NickLindeberg) [reply m4_sessions/1903_inning/automation.md:39]: We will add this to the list of questions.

File: m4_sessions/1903_inning/automation.md (hunk 7 above)
Comment (NickLindeberg) [reply m4_sessions/1903_inning/automation.md:41]: People are sharing in their small groups.

File: m4_sessions/1903_inning/automation.md (hunk 8 above)
Comment (NickLindeberg) [reply m4_sessions/1903_inning/automation.md:43]: Get one thought from each group.

**Pull request merged**